### PR TITLE
Count link redirects

### PR DESCRIPTION
**Pull request by turbaszek**
Related to #2

### CODE REVIEW

File: pkg/app/router.go [hunk 1]
```diff
@@ -92,6 +93,15 @@ func redirectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// To avoid redirect loop
+	if link.URL == link.ID {
+		http.Error(w, "Link id must be different than url", http.StatusBadRequest)
```
Comment (janusz-chludzinski) [inline pkg/app/router.go:98]: `Link id` is, IMHO, bit confusing. As a user I would probably confused what this ID is. As far as I understand now `id` is url part provided by the user? If so, I'd recommend changing it to something like `value` or something similar :)

**Review by janusz-chludzinski — APPROVED** [review]
👌 
One minor remark regarding naming things :)

File: pkg/app/router.go (hunk 1 above)
Comment (turbaszek) [reply pkg/app/router.go]: Good suggestion! I've renamed it and added some comments to structs

File: pkg/app/router.go [hunk 2]
```diff
@@ -92,6 +92,15 @@ func redirectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// To avoid redirect loop
+	if link.URL == link.Value {
+		http.Error(w, "Alternative url value must be different than the url", http.StatusBadRequest)
```
Comment (janusz-chludzinski) [inline pkg/app/router.go:97]: 👍

**Review by janusz-chludzinski — APPROVED** [review]
Nice one!

**Pull request merged**